tests/python/unittest/test_target_codegen_hexagon.py

```diff
@@ -107,7 +109,71 @@ def test_alloc_vtcm():
     assert "HexagonBackendFreeVTCM" in calls
 
 
+def test_linked_params_codegen():
+    if not check_prereq_and_setup():
```

Comment:
(also ok for a follow-up) do you mind migrating this to the pytest decorator style and modifying the `__main__` invocation:
```
if __name__ == "__main__":
    sys.exit(pytest.main([__file__] + sys.argv[1:]))
```

Reply:
Sure, I'll do it, but I'd prefer to do it in a separate PR, since I'd need to change some extra files.